files/lab/workshop/content/explore-openshift.md

```diff
@@ -16,21 +16,26 @@ Projects are a top level concept to help you organise your resources. An OpenShi
 
 OpenShift ships with a web-based console that will allow users to perform various tasks via a web browser. From within the lab guide window you'll see two buttons at the top of the screen, with the currently active tab in blue, one for the "**Terminal**", the default, and one for the "**Console**"; you can switch between these two options at will. Select the console and you should see the OpenShift web console, in which you'll already be logged in:
 
-<img  border="1" src="img/console-button.png"/>
+<img  border="1" src="img/console-button-new.png"/>
 
-**Entirely Optional**: However, if you want to open up an additional **dedicated** tab/window for the Web Console you can do so, simply click on [this link](http://console-openshift-console.%cluster_subdomain%/dashboards), although you will have to return to the instructions to get the "kubeadmin" (the OpenShift administrator user) password. To get the *kubeadmin-password* we'll need to grab it from the OpenShift installation directory on the server in which the cluster was installed from, or in our case the "bastion" node. Let's jump over to that machine to get it.
+With the Web Console provided in this lab environment, we will be able to execute the majority of the commands and we should be able to follow most of the lab sections. However, once we install the OpenShift Virtualization operator, this will add some "extra" graphical interfaces in the console that, unfortunately, cannot be displayed. But no need to worry, all we need to do is open a **dedicated** tab/window for the real full OpenShift Web Console.
+
+To do so, simply open a new window/tab in the browser you are using for this lab and navigate to this link: %cnvlab-console-url%. To login to the console, you will need to type "**kubeadmin**" (the OpenShift administrator user) as the user and paste its password: %cnvlab-kubeadmin-password%. 
+
+**Entirely Optional**: However, if you want, you can get the *kubeadmin-password* from the OpenShift installation directory on the server in which the cluster was installed from, or in our case the "bastion" node. Let's jump over to that machine to get it.
 
 From within this lab guide, SSH to the bastion node:
 
 ```execute-1
 ssh %bastion-username%@%bastion-host%
 ```
 
-When you see the prompt, if required, agree to the SSH certificates by typing "yes", and then enter **%bastion-password%** as the password. Then you can execute following command to get the kubeadmin password:
+When you see the prompt, if required, agree to the SSH certificates by typing "yes", and then enter the bastion password: %cnvlab-bastion-password%. Then you can execute the following command to get the kubeadmin password:
```

Comment:
this might not be needed anymore since we can reference the %cnvlab-kubeadmin-password% var directly now and this is not really a standard way to recover a lost openshift kubeadmin passwd either ;)

Reply:
Agree. Steps deleted in the new commit :)